app/uk/gov/hmrc/cipphonenumberverification/audit/VerificationCheckAuditEvent.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cipphonenumberverification.audit
+
+import play.api.libs.json.{JsObject, Json, Writes}
+
+case class VerificationCheckAuditEvent(phoneNumber: String, passcode: String, result: String) extends AuditEvent(phoneNumber, passcode)
+
+object VerificationCheckAuditEvent {
+  implicit val writes = new Writes[VerificationCheckAuditEvent] {
+    def writes(auditEvent: VerificationCheckAuditEvent): JsObject = Json.obj(
```

Comment:
Json.writes[VerificationCheckAuditEvent] should suffice. It is effectively shorthand for what you have here

Reply:
Done